code/game/gamemodes/changeling/powers/augmented_eyesight.dm

```diff
@@ -12,54 +12,25 @@
 /obj/effect/proc_holder/changeling/augmented_eyesight/sting_action(mob/living/carbon/human/user)
 	if(!istype(user))
 		return
-	if(user.getorgan(/obj/item/organ/cyberimp/eyes/thermals/ling))
-		user << "<span class='notice'>Our eyes are protected from flashes.</span>"
-		var/obj/item/organ/cyberimp/eyes/O = new /obj/item/organ/cyberimp/eyes/shield/ling()
-		O.Insert(user)
-
+	var/obj/item/organ/eyes/E = user.getorganslot("eye_sight")
+	if(E)
+		if(E.flash_protect)
+			E.sight_flags |= SEE_MOBS
```

Comment:
Even if they killed eachother during surgery, they're lings they could just res, are eyes removable from head items? if not they should be but that would also be a concern as like how robos just decap people for their brains, lings would just repeatedly decap eachother for mad eyeball profit.